app/models/recipe.rb

```diff
@@ -9,7 +9,6 @@ def self.call_api(uri)
         }
         return JSON.parse(res.body)
     end
-    @have_slept = false
```

Comment:
What was the purpose of this?

Reply:
forgot to remove this from testing when we were trying to overcome api throttling limits. sleeping function not needed anymore